src/OpenApi/Factory/OpenApiFactory.php

```diff
@@ -486,4 +511,25 @@ private function getSecuritySchemes(): array
 
         return $securitySchemes;
     }
+
+    private function appendSchemaDefinitions(\ArrayObject &$schemas, \ArrayObject $definitions): void
```

Comment:
Is the reference necessary?

Reply:
I think it's more performant like this indeed